staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -42,6 +42,7 @@ import (
 	"k8s.io/apiserver/pkg/features"
 	"k8s.io/apiserver/pkg/storage"
 	"k8s.io/apiserver/pkg/storage/cacher/metrics"
+	etcdfeature "k8s.io/apiserver/pkg/storage/etcd3/feature"
```

Comment:
I personally don't like the dependency from cacher/ into etcd3/
Should we actually move the feature package under /pkg/storage/feature instead?

@serathius

Reply:
Sounds good.

Reply:
@ah8ad3 - can you please move this?

Reply:
Yes. Will do it.